Generate.py

```diff
@@ -399,30 +399,91 @@ def roll_linked_options(weights: dict) -> dict:
     return weights
 
 
+def compare_results(
+        yaml_value: Union[str, int, bool, dict, list],
+        trigger_value: Union[str, int, bool, dict, list],
+        comparator: str):
+    if comparator == "=":
+        return yaml_value == trigger_value
+    elif comparator == "!=":
```

Comment:
There's a `return`, so this can be an `if`
```suggestion
    if comparator == "!=":
```